src/messages.ts

```diff
@@ -18,7 +18,9 @@ const _messages = {
     defaultValue: 'Enter default key value',
     addMissing: 'Add missing keys automatically?',
     missingValue: 'Missing value for',
-    done: 'Done!'
+    done: 'Done!',
+    problematicKeysForUnflat: (keys: string[]) =>
+      `The following keys won't be accessible when unflatting the object:\n    ${keys.join('\n    ')}`
```

Comment:
This can resolve in a pretty long output (because is key is printed in a new line), I think that this might be clearer:
```
The following keys won't be accessible when unflatting the object:\n ${keys.map(k => `"${k}"`).join(', ')}
```